SU-SWS/stanford_basic
# D8CORE-1313: Adding the big-paragraph mixin it the text paragraph list styles

# READY FOR REVIEW

# Summary
- added the mixin of big-paragraph to the ul/ol paragraph styles.

# Review By (Date)
- 2/21

# Urgency
- low

# Steps to Test

1. Pull in the branch.
2. Build and clear cache.
3. Add a text paragraph to a page and put in an ol and ul. Notice the spacing around the list is larger. 

# Affected Projects or Products
- Stanford Basic theme

# Associated Issues and/or People
- D8CORE-1313
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
